core/mutator/entry/mutator.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
I split the file.. not sure how git mv is supposed to handle this?  
I could have tried git cp?